package.json

```diff
@@ -1,19 +1,35 @@
 {
   "name": "polymer-project-builder",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "Custom Polymer project builder",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --coverage",
+    "test:watch": "jest --coverage --watch",
+    "lint:js": "eslint --max-warnings 0 ./*.js",
+    "commit": "git-cz",
+    "semantic-release": "semantic-release"
   },
   "keywords": [
     "cli",
     "polymer"
   ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/BBVAEngineering/polymer-project-builder.git"
+  },
   "author": "",
   "license": "MIT",
+  "bin": {
+    "build-polymer-project": "bin/cli.js"
+  },
   "dependencies": {
```

Comment:
Use `dependencies` for real dependencies and `devDependencies` for development dependencies.